lib/ClangImporter/ClangImporter.cpp

```diff
@@ -424,6 +424,17 @@ getNormalInvocationArguments(std::vector<std::string> &invocationArgStrs,
     });
   }
 
+  // If there are no shims in the resource dir, add a search path in the SDK.
+  SmallString<128> shimsPath(searchPathOpts.RuntimeResourcePath);
+  llvm::sys::path::append(shimsPath, "shims");
+  if (!llvm::sys::fs::exists(shimsPath)) {
+    shimsPath = searchPathOpts.SDKPath;
+    llvm::sys::path::append(shimsPath, "usr", "lib", "swift", "shims");
+    invocationArgStrs.insert(invocationArgStrs.end(), {
+      "-I", shimsPath.str()
```

Comment:
This should probably be consistent about -I vs. -isystem (cf. https://github.com/apple/swift/blob/a20d67179ab77f22e370bdb1543e890917dc7bc2/lib/ClangImporter/ClangImporter.cpp#L451)